tests/test_projects.py

```diff
@@ -87,6 +87,7 @@ def test_upload_file_to_project(default_domino_client):
     assert response.status_code == 201
     assert response.json()["path"] == "test_file.py"
 
+
```

Comment:
why the extra line?

Reply:
that's python standard spacing between functions.

Reply:
that makes sense to add, but it doesn't seem related to any of the other changes

I would maybe have a separate pull request for it or wait until that file needs to be changed

Reply:
that's a black - pre-commit fix. if the file had particularly issues, once touched, blacks updates.

Reply:
I see, ok that makes sense to keep then